test.cmd

```diff
@@ -0,0 +1,24 @@
+@echo off
+setlocal
+cd "%~dp0"
+chcp 1252 > nul
+if not exist tools\NUnit.Runners (
+    for %%i in (NuGet.exe) do set nuget=%%~dpnx$PATH:i
+    if "%nuget%"=="" goto :nonuget
+    md tools
+    nuget.exe install NUnit.Runners -Version 2.6.4 -OutputDirectory tools
```

Comment:
Missing check if `nuget install` failed or not. Perhaps use something like this?

```cmd
nuget.exe install NUnit.Runners -Version 2.6.4 -OutputDirectory tools || exit /b 1
```